plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -64,6 +66,53 @@ public static boolean hasFacet(IFacetedProject project) {
   }
 
   /**
+   * Return the App Engine standard facet for the given project, or {@code null} if none.
+   */
+  public static IProjectFacetVersion getProjectFacetVersion(IProject project) {
+    try {
+      IFacetedProject facetedProject = ProjectFacetsManager.create(project);
+      if (facetedProject == null) {
+        return null;
+      }
+      IProjectFacet projectFacet = ProjectFacetsManager.getProjectFacet(ID);
+      return facetedProject.getProjectFacetVersion(projectFacet);
+    } catch (CoreException ex) {
+      return null;
+    }
+  }
+
+  /**
+   * Check that the given Servlet API version string (expected to be from the
+   * <tt>&lt;web-app version="xxx"&gt;</tt> attribute), is compatible with this project's App Engine
+   * settings.
+   * 
+   * @return {@code true} if supported or {@code false} otherwise
+   */
+  public static boolean checkServletApiSupport(IProject project, String servletApiVersion) {
```

Comment:
I think it's straightforward to test this whole method, since you already have projects created from `TestProjectCreator`. Any reason for breaking down the method to test part of it?